### DISCUSSION TITLE
How can I use this with script tag?

### DISCUSSION THREAD

**Discussion by marcusx2** [pdfme/pdfme#455] [Q&A]
I want to use pdfme by importing it with a [script](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) tag. I don't use nodejs, typescript, webpack, etc.

I want to import pdfme with the script tag like `pdfjs`

```
<script type="module">
import pdfjsDist from 'https://cdn.jsdelivr.net/npm/pdfjs-dist@4.0.379/+esm'
</script>
```

or `pdf-lib`
`<script src="https://unpkg.com/pdf-lib"></script>`

How can I do this?

**Comment by peteward** [comment]
Hi @marcusx2,

For a good example of embedding pdfme within an existing application see the [playground designer example](https://github.com/pdfme/pdfme/blob/main/playground/src/Designer.tsx).

**Comment by marcusx2** [comment]
I don't get it. What's these .tsx files? I want to make it work with an index.html and .js file, nothing else. Not using nodejs, typescript, webpack or any fancy stuff.

**Comment by marcusx2** [comment]
How do I bundle this into a single .js file (or multiple)? I'm trying to use Webpack but I'm a noob at this. Can someone give me detailed instructions? Thanks!

EDIT

I saw the [development guide](https://github.com/pdfme/pdfme/blob/main/DEVELOPMENT.md), but how do I actually import pdfme on a single index.html file with the script tag? Thanks!

**Comment by Meadril** [comment]
@marcusx2 go to this site, get your links to the libraries and then include them using script tags.

https://www.jsdelivr.com/?query=pdfme

Like `<script src="https://cdn.jsdelivr.net/npm/@pdfme/common@4.3.2/dist/cjs/src/index.min.js"></script>`
